package.json

```diff
@@ -25,6 +25,7 @@
   "devDependencies": {
     "broccoli-asset-rev": "^2.4.5",
     "ember-ajax": "^2.4.1",
+    "ember-buffered-proxy": "^0.7.0",
```

Comment:
You will need to update yarn.lock with this package change

Reply:
Thanks for the reminder, seems `ember install xxx` won't update the yarn.lock.

Reply:
If I'm not mistaken its because this project is still on ember-cli 2.12 which doesn't include the yarn stuff so we might want to update that as well at some point

Reply:
True, yarn support was added in 2.13. 
https://www.emberjs.com/blog/2017/03/19/ember-2-12-released.html#toc_add-support-for-using-code-yarn-code